packages/gappa/gappa.1.3.5/opam

```diff
@@ -40,13 +40,6 @@ depends: [
 ]
 synopsis: "Tool intended for formally proving properties on numerical programs dealing with floating-point or fixed-point arithmetic"
 url {
-  src: "https://gitlab.inria.fr/gappa/gappa/-/archive/gappa-1.3.5.tar.gz"
-  checksum: "sha512=29ce59af97e6d60547a193b43538f4812ff74fb01a812cda7109855219457fa7a47f59ea39aff2a5e03fd70181e024a3296b4f48300818a81f62fd2d8629c389"
+  src: "https://gappa.gitlabpages.inria.fr/releases/gappa-1.3.5.tar.gz"
+  checksum: "sha512=60b5719e3a321df43e33045fa8f4511fc02a4218d1ae7e476e7c6ebcf90ae208832881f6eea5b99a3296dfcc3a18c7e1f4ea9dbea446fc502e14306b6975f6e6"
 }
-extra-files: [
```

Comment:
Why are these gone? Have they been integrated in the new tarballs?

We are reluctant to tarball changes in the repository nowadays (See https://github.com/ocaml/opam-repository/wiki/Policies#11-changes-to-a-packages-source-archive-are-prohibited).

Before merging this, I need to obtain the old tarballs and manually verify that there was no change between the old and the new. If possible, I'd reinstate the old tarballs at the new urls, If you don't have them, you can likely find them following https://github.com/ocaml/opam-repository/wiki/FAQ#how-to-find-lost-archives-of-packages

Reply:
@mseri : I found that the extra_files section is not required for patch files - I guess opam trusts them as much as the opam file if they come from the same source, so its doesn't require a checksum. So I tend to remove them when I clean up things.

The old tarballs are still there - both new and old locations are OK. The newer versions are the same. The version 1.3.5 defers, but it should be a minor difference. I have seen this in the past that on the fly generated tar balls change their checksum.

Anyway according to the author (Guillaume Melquiond) the new locations are the official locations. See also https://gappa.gitlabpages.inria.fr/releases/.

Reply:
For the old releases, I'd rather use the old tarballs. If they cannot be replaced on the new locations, I suggest to upload them on the opam-source-archives repository and link them from there